### ISSUE TITLE
binary_cross_entropy_with_logits crashes on Colab

### ISSUE THREAD

**Issue by henrique** [pytorch/xla#1660]
## 🐛 Bug

I've converted the CIFAR example into a binary classification
but it crashes at the loss function (F.binary_cross_entropy_with_logits):
```
Exception in device=TPU:0: torch_xla/csrc/aten_xla_bridge.cpp:62 : Check failed: xtensor 
*** Begin stack trace ***
	tensorflow::CurrentStackTrace[abi:cxx11]()
	torch_xla::bridge::GetXlaTensor(at::Tensor const&)
	torch_xla::AtenXlaType::mul_(at::Tensor&, at::Tensor const&)
	
	at::native::binary_cross_entropy_with_logits(at::Tensor const&, at::Tensor const&, at::Tensor const&, at::Tensor const&, long)
	torch_xla::AtenXlaType::binary_cross_entropy_with_logits(at::Tensor const&, at::Tensor const&, at::Tensor const&, at::Tensor const&, long)
	c10::detail::wrap_kernel_functor_unboxed_<c10::detail::WrapRuntimeKernelFunctor_<at::Tensor (*)(at::Tensor const&, at::Tensor const&, at::Tensor const&, at::Tensor const&, long), at::Tensor, c10::guts::typelist::typelist<at::Tensor const&, at::Tensor const&, at::Tensor const&, at::Tensor const&, long> >, at::Tensor (at::Tensor const&, at::Tensor const&, at::Tensor const&, at::Tensor const&, long)>::call(c10::OperatorKernel*, at::Tensor const&, at::Tensor const&, at::Tensor const&, at::Tensor const&, long)
	_PyEval_EvalFrameDefault
```
(...)

## Environment

 - Colab as in https://colab.research.google.com/github/pytorch/xla/blob/master/contrib/colab/resnet18-training-xrt-1-15.ipynb

Thanks

**Comment by henrique** [comment]
the msg doesn't explain it, but the target tensor wasn't properly copied to XLA
sorry for the false alarm and thanks for the awesome work
